src/main/java/com/climatetree/places/controller/PlacesController.java

```diff
@@ -9,13 +9,15 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 import com.climatetree.places.dto.PlaceDTO;
 import com.climatetree.places.service.NamesService;
 import com.climatetree.places.service.PlacesService;
 
 @RestController
+@RequestMapping("/api")
```

Comment:
You'll have to update the controller tests to include "api" in the mapping for tests to pass